src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsService.java

```diff
@@ -14,14 +20,41 @@
 public class AssessmentsService {
 
   private final InstallmentNoPIIService installmentNoPIIService;
+  private final IngestionFlowFileService ingestionFlowFileService;
+  private final DebtPositionTypeOrgService debtPositionTypeOrgService;
+  private final AssessmentsRepository assessmentsRepository;
 
-
-  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService) {
+  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService, IngestionFlowFileService ingestionFlowFileService, DebtPositionTypeOrgService debtPositionTypeOrgService, AssessmentsRepository assessmentsRepository) {
     this.installmentNoPIIService = installmentNoPIIService;
+    this.ingestionFlowFileService = ingestionFlowFileService;
+    this.debtPositionTypeOrgService = debtPositionTypeOrgService;
+    this.assessmentsRepository = assessmentsRepository;
   }
 
   public List<InstallmentNoPIIResponse> getInstallmentsByReceiptId(Long receiptId, String accessToken) {
     return installmentNoPIIService.getByReceiptId(receiptId, accessToken);
   }
 
+  public Assessments getAssessment(InstallmentNoPIIResponse installmentNoPIIResponse, String accessToken) {
+    IngestionFlowFile ingestionFlowFile = ingestionFlowFileService.getIngestionFlowFile(installmentNoPIIResponse.getIngestionFlowFileId(), accessToken);
```

Comment:
why are you fetching the IngestionFlowFile? The organization should be retrieved from the DebtPosition, not from the IngestionFlowFile... if the Installment has not been created with a massive ingestion, this column will be null.

The organizationId could be read from the debtPositionTypeOrg entity instead